bot/exts/info/codeblock/_instructions.py

```diff
@@ -177,7 +164,4 @@ def get_instructions(content: str) -> str | None:
             if not instructions:
                 instructions = _get_no_lang_message(block.content)
 
-    if instructions:
-        instructions += "\nYou can **edit your original message** to correct your code block."
```

Comment:
Do you think this line was ineffective? The intent was to avoid spamming the channel with several attempts to fix their message. I don't know a better way to communicate this.

Reply:
I moved this instruction to the title for the embed.

Reply:
Oh I need to read more carefully, sorry!

Reply:
No problem; thank you for having written the original code in such a way that the changes in this PR were straightforward to make.